components/ActivityIndicator/examples/ActivityIndicatorTransitionExample.m

```diff
@@ -1,12 +1,12 @@
 /*
  Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
- 
+
  Licensed under the Apache License, Version 2.0 (the "License");
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at
- 
+
  http://www.apache.org/licenses/LICENSE-2.0
- 
+
```

Comment:
I typically push back against *any* changes to our license stanza once they've been published.